### PR TITLE
Add scroll left/right to ScrollRobot

### LINKED ISSUE #283: ScrollRobot should support horizontal directions
`org.testfx.robot.ScrollRobot` only supports scrolling up and down.

### PR DESCRIPTION
**Pull request by aalmiray**
Add support for horizontal scrolling to ScrollRobot and FxRobot.

Fixes #283

### CODE REVIEW

File: subprojects/testfx-core/src/main/java/org/testfx/robot/ScrollRobot.java [hunk 1]
```diff
@@ -16,17 +16,24 @@
...
                        VerticalDirection direction);
 
-    public void scrollUp(int positiveAmount);
+    void scrollUp(int positiveAmount);
 
-    public void scrollDown(int positiveAmount);
+    void scrollDown(int positiveAmount);
 
+    void scroll(int positiveAmount,
+                       HorizontalDirection direction);
```
Comment (brcolow) [inline subprojects/testfx-core/src/main/java/org/testfx/robot/ScrollRobot.java]: Weird indentation on the second argument here.

Comment (aalmiray) [inline subprojects/testfx-core/src/main/java/org/testfx/robot/ScrollRobot.java]: Fixed with https://github.com/TestFX/TestFX/pull/285/commits/9fb9674fb4b8250f658107d4b37dc4f8ee751b4a

**Comment by brcolow** [comment]
Merged. Thanks a lot!

**Comment by aalmiray** [comment]
lovely! Eagerly awaiting the next release 😄

**Comment by JordanMartinez** [comment]
@aalmiray I am adding documentation in #325 and I came across your work here. When I looked at the the actual code that gets called in `GlassAdapterRobot`, I don't understand how the system is able to distinguish a scroll up from a scroll left or a scroll down from a scroll right. Care to explain?

**Comment by aalmiray** [comment]
The original implementation of the scroll methods simply incremented the current position by 1. This would scroll to the right if the orientation is horizontal, down if vertical. The new ScrollRobot class adds left/up support by  the position by -1.

**Comment by JordanMartinez** [comment]
Ah... so orientation is what determines that... I didn't understand how passing a value of 1 could scroll both down and right.

**Pull request merged**